spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-function/src/main/java/org/springframework/cloud/fn/tasklauncher/LaunchRequest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.fn.tasklauncher;
```

Comment:
Package names should somewhat follow module name within dataflow(spring-cloud-dataflow-composed-task-runner which were also moved in). Maybe `org.springframework.cloud.dataflow.tasklauncher`.